calico_node/tests/st/bgp/test_backends.py

```diff
@@ -56,7 +56,7 @@ def test_bgp_backends(self):
             workload_host3 = host3.create_workload("workload3", network=network1, ip=DEFAULT_IPV4_ADDR_3)
 
             # Allow network to converge
-            self.assert_true(workload_host1.check_can_ping(workload_host2.ip, retries=10))
+            self.assert_true(workload_host1.check_can_ping(workload_host2.ip, retries=15))
```

Comment:
No idea - these should not need to be updated for this PR.  Perhaps local environment is slower?